src/ucp/api/ucp.h

```diff
@@ -1878,6 +1981,85 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
 
 /**
  * @ingroup UCP_COMM
+ * @brief Non-blocking stream receive operation.
+ *
+ * This routine receives a data that is described by the local address @a
+ * buffer, size @a count, and @a datatype object on the endpoint @a ep. The
+ * routine is a non-blocking and therefore returns immediately. The receive
+ * operation is considered completed when the message is delivered to the @a
+ * buffer. In order to notify the application about completion of the receive
+ * operation the UCP library will invoke the call-back @a cb when the received
+ * data is in the receive buffer and ready for application access. If the
+ * receive operation cannot be started the routine returns an error.
```

Comment:
I suggest rewording the comment block as follows:
This routine receives data that is described by the local address  _buffer_, size  _count_, and  _datatype_ object on the endpoint  _ep_. The routine is non-blocking and therefore returns immediately. The receive operation is considered
complete when the message is delivered to the _buffer_. If data is not immediately available, the operation will be
scheduled for receive and a request handle will be returned. In order to notify the application about completion of
a scheduled receive operation, the UCP library will invoke the call-back cb when data is in the receive buffer and
ready for application access. If the receive operation cannot be started, the routine returns an error.